docs/fields-gen.asciidoc

```diff
@@ -1,23 +1,28 @@
 [[ecs-base]]
 === Base fields
 
-The base set contains all fields which are on the top level. These fields are common across all types of events.
+The base set contains top level fields that are common across all types of events
+(such as `@timestamp`, `tags`, `message`, and `labels`).
 
-[cols="<,<,<,<,<",options="header",]
+[cols="<,<,<",options="header",]
 |=======================================================================
-| Field  | Description  | Level  | Type  | Example 
-| [[@timestamp]] | Date/time when the event originated. 
+| Field  | Description  | Level 
+| [[@timestamp]] | `date`. Date/time when the event originated. 
 For log events this is the date/time when the event was generated, and not when it was read.
-Required field for all events. | core | date | `2016-05-23T08:05:34.853Z` 
-| tags | List of keywords used to tag each event. | core | keyword | `["production", "env2"]` 
-| labels | Key/value pairs.
+Required field for all events. Example: `2016-05-23T08:05:34.853Z` | core 
```

Comment:
Could we try introducing a line break, and mention the data type above or below the Level, in the cells?

Not sure how we'll name the column for this hybrid column, but I wouldn't worry about that just yet. I'm just curious to see if this renders well, and if it makes sense visually.

```suggestion
Required field for all events. Example: `2016-05-23T08:05:34.853Z` | core 
date
```

Reply:
See the new approach in the next commit.  I think you'll like it better than what we have here.